src/qibosoq/rfsoc_server.py

```diff
@@ -49,14 +49,14 @@ def execute_program(self, data: dict) -> dict:
         Returns:
             (dict): dictionary with two keys (i, q) to lists of values
         """
-        if data["operation_code"] == "execute_pulse_sequence":
+        if OperationCode(data["operation_code"]) is OperationCode.EXECUTE_PULSE_SEQUENCE:
```

Comment:
```suggestion
        opcode = OperationCode(data["operation_code"])
        if opcode is OperationCode.EXECUTE_PULSE_SEQUENCE:
```

Just to save the repeated operation.